<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdaterWindow</class>
 <widget class="QMainWindow" name="UpdaterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Software Update</string>
  </property>
  <widget class="QWidget" name="Outer">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="newVersionIsAvailableLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>A new version of %1 is available!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="wouldYouLikeToDownloadLabel">
      <property name="text">
       <string>%1 %2 is now available - you have %3. 
Would you like to download and install it?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="releaseNotes">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="skipThisVersionButton">
        <property name="text">
         <string>Skip This Version</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="UpdateDownloadProgress" name="updateDownloadProgress" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remindMeLaterButton">
        <property name="text">
         <string>Remind Me Later</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="installUpdateButton">
        <property name="text">
         <string>Install Update</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UpdateDownloadProgress</class>
   <extends>QWidget</extends>
   <header>Partials/UpdateDownloadProgress.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
